builtin/settingtypes.txt

```diff
@@ -71,7 +71,7 @@ fast_move (Fast movement) bool false
 noclip (Noclip) bool false
 
 #    Smooths camera when moving and looking around.
-#    Useful for recording videos.
+#    Useful for recording videos. Aka mouse or look smoothing.
```

Comment:
I feel 'aka' should be avoided as it's probably unclear to many internationally.
'look smoothing' seems redundant since we have 'Smooths camera when moving and looking around'